src/tools/s2wasm.cpp

```diff
@@ -33,6 +34,7 @@ int main(int argc, const char *argv[]) {
   bool generateEmscriptenGlue = false;
   std::string startFunction;
   std::vector<std::string> archiveLibraries;
+  bool validate = false;
```

Comment:
let's not make this optional. we don't have it optional in other parallel places.